lightning/web/progress.go

```diff
@@ -184,3 +184,10 @@ func MarshalTaskProgress() ([]byte, error) {
 func MarshalTableCheckpoints(tableName string) ([]byte, error) {
 	return currentProgress.checkpoints.marshal(tableName)
 }
+
+func DisplayCheckPointStatus() bool {
+	if currentProgress.Tables == nil {
+		return true
+	}
+	return false
+}
```

Comment:
1. Please choose a better name... This method is not "displaying" anything.
2. `return currentProgress.Tables == nil`.